_studio/shared/asc/include/asc_defs.h

```diff
@@ -55,10 +55,14 @@
 #define SCD_BLOCK_PIXEL_WIDTH   32
 #define SCD_BLOCK_HEIGHT        8
 
-#ifdef _DEBUG
+#if defined(_DEBUG)
     #define ASC_PRINTF(...)     printf(__VA_ARGS__)
     #define ASC_FPRINTF(...)    fprintf(__VA_ARGS__)
     #define ASC_FFLUSH(x)       fflush(x)
+#elif defined(ANDROID)
+	#define ASC_PRINTF(...)
+    #define ASC_FPRINTF(...)
+    #define ASC_FFLUSH(x)
 #else
```

Comment:
Even in DEBUG mode we should not print anything to console in the library. I.e. This code does not make ANY sense actually. This whole block should be rewritten at least as:
```
//#define ASC_DEBUG

#if defined(ASC_DEBUG)
    #define ASC_PRINTF(...)     printf(__VA_ARGS__)
    #define ASC_FPRINTF(...)     fprintf(__VA_ARGS__)
    #define ASC_FFLUSH(x)    fflush(x)
#else
    #define ASC_PRINTF(...)
    #define ASC_FPRINTF(...)
    #define ASC_FFLUSH(x)
#endif
```
Pay attention that ASC_DEBUG is commented out!

Reply:
Should this change be made in closed source?